free-1.md

```diff
@@ -57,12 +73,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The developer's attentions would probably be distracted by creating flavor pairings and figuring out a method of efficient searching through a connected graph. It would get particularly confusing to introduce categories (potentially in the form of attribute tags) for each different flavor or food. It would be easy to spend too much time focusing on this setup instead of the actual language for the user.
```

Comment:
I agree with this. Going back to what I said about a DSL compared to an application, this seems more like something that's mostly concerned with the systems aspect rather than the language aspects. I also found several of the ideas I came up with ended up being more about the system than the language. Now that I think about it, it's kind of hard to find a good DSL project idea...